config/default.json

```diff
@@ -336,7 +336,7 @@
         "LoginButtonTextColor": ""
     },
     "NativeAppSettings": {
-        "AppDownloadLink": "https://about.mattermost.com/downloads/",
+        "AppDownloadLink": "https://mattermost.com/download/#mobile",
```

Comment:
Were we wanting to use a redirect for this in case the download page changes in the future? Something like sending them to `https://about.mattermost.com/mattermost-apps` which then sends the user to `https://mattermost.com/download/#mobile` I thought we did something like that for the links in SupportSettings, but I might be incorrect there

Reply:
Okay thanks Harrison, I asked Rich about creating a redirect

Reply:
@esethna What is the status of this PR? Based on the ticket this doesn't seem to be a regression, so optionally this can be pushed to v5.7

Reply:
I don't think the web team got to this yet, we can push to next release